documentation/modules/security/proc-installing-your-own-ca-certificates.adoc

```diff
@@ -36,73 +38,38 @@ The chain should be in the following order:
-
-* `ca.crt` cluster certificate in PEM format
-* `ca.p12` cluster certificate in PKCS #12 format
-* `ca.password` to access the PKCS #12 file
+The Cluster Operator generates keys and certificates in PEM (Privacy Enhanced Mail) and PKCS #12 (Public-Key Cryptography Standards) formats.
+You can add your own certificates in either format.  
 
 Some applications cannot use PEM certificates and support only PKCS #12 certificates.
-You can also add your own cluster certificate in PKCS #12 format.
-
 If you don't have a cluster certificate in PKCS #12 format, use the OpenSSL TLS management tool to generate one from your `ca.crt` file.
 
 .Example certificate generation command
 [source,shell,subs="+quotes"]
-openssl pkcs12 -export -in ca.crt -nokeys -out ca.p12 -password pass:__P12-PASSWORD__ -caname ca.crt
-
-Replace _P12-PASSWORD_ with your own password.
+openssl pkcs12 -export -in ca.crt -nokeys -out ca.p12 -password pass:__<P12_password>__ -caname ca.crt
 
-You can do the same for the `_CLUSTER-NAME_-clients-ca-cert` secret, which also contains certificates in PEM and PKCS #12 format by default.
+Replace _<P12_password>_ with your own password.
 
 .Procedure
 
-. Replace the CA certificate generated by the Cluster Operator.
-
-.. Delete the existing secret.
-+
-[source,shell,subs="+quotes"]
-kubectl delete secret _CA-CERTIFICATE-SECRET_
-+
-_CA-CERTIFICATE-SECRET_ is the name of the `Secret`:
-+
---
-* `_CLUSTER-NAME_-cluster-ca-cert` for the cluster CA certificate
-* `_CLUSTER-NAME_-clients-ca-cert` for the clients CA certificate
---
-+
-Replace _CLUSTER-NAME_ with the name of your Kafka cluster.
-+
-Ignore any "Not Exists" errors.
-
-.. Create the new secret.
+. Create a new secret to replace the CA certificate generated by the Cluster Operator.
 +
 .Client secret creation with a certificate in PEM format only
 [source,shell,subs="+quotes"]
-kubectl create secret generic _CLUSTER-NAME_-clients-ca-cert --from-file=ca.crt=ca.crt
+kubectl create secret generic _<cluster_name>_-clients-ca-cert --from-file=ca.crt=ca.crt
 +
 .Cluster secret creation with certificates in PEM and PKCS #12 format
 [source,shell,subs="+quotes"]
 ----
-kubectl create secret generic _CLUSTER-NAME_-cluster-ca-cert \
+kubectl create secret generic _<cluster_name>_-cluster-ca-cert \
   --from-file=ca.crt=ca.crt \
   --from-file=ca.p12=ca.p12 \
   --from-literal=ca.password=_P12-PASSWORD_
 ----
-
-. Replace the private key generated by the Cluster Operator.
-+
-.. Delete the existing secret.
-+
-[source,shell,subs="+quotes"]
-kubectl delete secret _CA-KEY-SECRET_
-+
-_CA-KEY-SECRET_ is the name of CA key:
 +
-* `_CLUSTER-NAME_-cluster-ca` for the cluster CA key
-* `_CLUSTER-NAME_-clients-ca` for the clients CA key
+Replace _<cluster_name>_ with the name of your Kafka cluster.
 
-.. Create the new secret.
+. Create a new secret to replace the private key generated by the Cluster Operator.
```

Comment:
Isn't this procedure about installing your own CA cert + key from scratch with cluster not running? so we are not replacing, are't we?

Reply:
Yes. That could be misread. I've changed as follows:

> 1. Create a new secret that contains the CA certificate.
>  2. Create a new secret that contains the private key.